apps/ml/site/project/projects.jag

```diff
@@ -663,6 +695,51 @@ include("../../includes/tenantAware.jag");
         return notification;
     }
 
+    function getAvailableDatasetCount(){
+        var count;
+        $.ajax({
+            type: "GET",
+            url: serverUrl+"/api/datasets?status=Available",
+            async: false,
+            success: function(data) {
+                if(data.length == 0){              
+                    count =0;
+                }
+                else{

```

Comment:
Formatting issue. "else" statement should start right after the "if" closing curly brackets.